README.md

```diff
@@ -29,33 +29,25 @@ The samples are either focused on a single azure service (**Single Tech Samples*
 
 ## Single Technology Samples
 
-- [Microsoft Fabric](./single_tech_samples/fabric/README.md)
-  - [CI/CD - Microsoft Fabric](./single_tech_samples/fabric/fabric_ci_cd/README.md)
-  - [Feature engineering on Microsoft Fabric](./single_tech_samples/fabric/feature_engineering_on_fabric/README.md)
-- [Azure SQL database](./single_tech_samples/azuresql/README.md)
-  - [CI/CD - Azure SQL database](./single_tech_samples/azuresql/azuresql_ci_cd/README.md)
-- [Azure Databricks](single_tech_samples/databricks/)
-  - [IaC - Basic deployment](single_tech_samples/databricks/databricks_ci_cd/README.md)
-- [Azure Data Factory](./single_tech_samples/datafactory/README.md)
-  - [CI/CD - Auto publish](./single_tech_samples/datafactory/adf_cicd_auto_publish/README.md)
-  - [Data pre-processing using Azure Batch](./single_tech_samples/datafactory/adf_data_pre_processing_with_azure_batch/README.md)
-- [Azure Synapse Analytics](./single_tech_samples/synapseanalytics/README.md)
-  - [Serverless best practices](./single_tech_samples/synapseanalytics/synapse_serverless/README.md)
-- [Azure Stream Analytics](./single_tech_samples/streamanalytics/README.md)
-  - [CI/CD - Azure Stream Analytics](./single_tech_samples/streamanalytics/streamanalytics_ci_cd/README.md)
+| Technology | Samples |
+| ---------- | ------- |
+| [Microsoft Fabric](./single_tech_samples/fabric/README.md) | ▪️ [CI/CD for Microsoft Fabric](./single_tech_samples/fabric/fabric_ci_cd/README.md)<br> ▪️ [Feature engineering on Microsoft Fabric](./single_tech_samples/fabric/feature_engineering_on_fabric/README.md) |
+| [Azure SQL database](./single_tech_samples/azuresql/README.md) | ▪️ [CI/CD for Azure SQL database](./single_tech_samples/azuresql/azuresql_ci_cd/README.md) |
+| [Azure Databricks](single_tech_samples/databricks/) | ▪️ [CI/CD for Azure Databricks](single_tech_samples/databricks/databricks_ci_cd/README.md) |
+| [Azure Data Factory](./single_tech_samples/datafactory/README.md) | ▪️ [CI/CD for ADF with Auto publish](./single_tech_samples/datafactory/adf_cicd_auto_publish/README.md)<br> ▪️ [Data pre-processing using Azure Batch](./single_tech_samples/datafactory/adf_data_pre_processing_with_azure_batch/README.md) |
+| [Azure Stream Analytics](./single_tech_samples/streamanalytics/README.md) | ▪️ [CI/CD for Azure Stream Analytics](./single_tech_samples/streamanalytics/streamanalytics_ci_cd/README.md) |
 
 ## End to End samples
 
-### Parking Sensor Solution
+### DataOps for Medallion with Azure Data Factory and Azure Databricks
 
-This demonstrates batch, end-to-end data pipeline following the MDW architecture, along with a corresponding CI/CD process.
+This [sample](e2e_samples/parking_sensors/) demonstrates batch, end-to-end data pipeline utilizing Azure Data Factory and Azure Databricks built according to the Medallion architecture, along with a corresponding CI/CD process, Observability and automated testing.
```

Comment:
We can explain in 2-3 sentences what Medallion architecture is.
or
At least, provide hyperlink to the doco:
[Medallion architecture](https://learn.microsoft.com/en-us/azure/databricks/lakehouse/medallion)

Reply:
Added